### PR TITLE
added a long version

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
divide text to a short and a long version to correct appearance in list and detail views

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13117

## Screenshots
Simulator Screen Shot - iPhone 13 Pro - 2022-05-23 at 14 12 38
Simulator Screen Shot - iPhone 13 Pro - 2022-05-23 at 14 12 42

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCell.swift [hunk 1]
```diff
@@ -118,6 +118,7 @@ class HealthCertificateQRCodeCell: UITableViewCell, ReuseIdentifierProviding {
 		let validityStateStackView = UIStackView()
 		validityStateStackView.alignment = .center
 		validityStateStackView.axis = .horizontal
+		validityStateStackView.distribution = .fillProportionally
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCell.swift:121]: `fillProportionally` in combination with a fixed-size image doesn't look quite right to me, did the default `fill` not work? If not, maybe some hugging/content compression priorities were not set correctly?

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonCellModel.swift [hunk 2]
```diff
@@ -142,7 +142,7 @@ class HealthCertifiedPersonCellModel {
 			return nil
 		case .expiringSoon:
 			let validityStateTitle = String(
-				format: AppStrings.HealthCertificate.ValidityState.expiringSoon,
+				format: AppStrings.HealthCertificate.ValidityState.expiringSoonShort,
```
[the remark this answers is not in the document]
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonCellModel.swift:145]: I will discuss it with Anna-Maria

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCell.swift (hunk 1 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCell.swift:121]: `.fill` works fine too - I changed it

**Review by nickguendling — APPROVED** [review]

**Pull request merged**